botfront/imports/ui/components/stories/StoryFooter.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Segment, Breadcrumb, Icon } from 'semantic-ui-react';
+
+import './style.import.less';
+
+import StoryPathPopup from './StoryPathPopup.jsx';
+
+class StoryFooter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    renderPath = () => {
+        const { storyPath } = this.props;
+        let pathArray = storyPath.split('__');
+        const pathBreadcrumbs = [];
+        const maxLength = 5;
+
+        if (pathArray.length > 3) {
+            pathBreadcrumbs.push();
+            pathArray = pathArray.slice(pathArray.length - maxLength, pathArray.length);
+            pathBreadcrumbs.push(
+                <StoryPathPopup
+                    storyPath={storyPath}
+                    trigger={(
+                        <Breadcrumb.Section className='collapsed-path'>
+                            <Icon disabled color='grey' name='ellipsis horizontal' size='large' />
+                        </Breadcrumb.Section>
+                    )}
+                />,
+            );
+            pathBreadcrumbs.push(<Breadcrumb.Divider>{'>'}</Breadcrumb.Divider>);
+        }
+        pathArray.forEach((location) => {
+            pathBreadcrumbs.push(<Breadcrumb.Section>{location}</Breadcrumb.Section>);
+            pathBreadcrumbs.push(<Breadcrumb.Divider>{'>'}</Breadcrumb.Divider>);
+        });
+        pathBreadcrumbs.pop();
+        return pathBreadcrumbs;
+    };
+
+    handleBranchClick = () => {
+        const { onBranch, canBranch } = this.props;
+        if (canBranch) {
+            onBranch();
+        }
+    };
+
+    handlerContinueClick = () => {
+        onContinue();
+    };
+
+    selectIconColor = (active) => {
+        if (active) {
+            return 'blue';
+        }
+        return 'grey';
+    }
+
+    renderContinue = () => {
+        const { canContinue, disableContinue } = this.props;
+        if (disableContinue) {
+            return <></>;
+        }
+        if (canContinue) {
+            return (
+                <Segment className='footer-option-button' onClick={this.handlerContinueClick}>
```

Comment:
Please make these more button-like (at the minimum with a pointer on hover, an active color, etc)